docs/DeveloperGenerateTest.md

```diff
@@ -0,0 +1,91 @@
+# Generating Tests
+
+Before generating tests you should be aware of the following environment variables that are used to render new test files from a template:
+
+| Variable | Required | Explanation | Default Value |
+|---|---|---|---|
+| `TEMPLATE_SUITE_PACKAGE_NAME` | yes | The name of the package directory and spec file created in `tests/` and the name used when creating the suite file in `cmd/`  | `template`  |
```

Comment:
nit: this var has a default value specified, so is it really required?

Reply:
Good point. I think I defaulted to `template` to just get started but I don't foresee anyone actually sticking with the name `template` when actually submitting a PR to prod. I would assume they would rename the files and specs files when they are ready to productize the tests. I will remove the default value and make it explicit